recipe/meta.yaml

```diff
@@ -44,10 +56,26 @@ test:
     - tokenizers.processors
     - tokenizers.trainers
     - tokenizers.implementations
-  commands:
-    - pip check
   requires:
     - pip
+    - pytest
+    - datasets
+    - numpy *
```

Comment:
I hadn't seen the `*` notation in a recipe, TIL :).

Reply:
It keeps conda smithy from spuriously thinking it needs to compile the recipe against different numpy versions. :)

Reply:
Neat!